src/spam-checker.js

```diff
@@ -0,0 +1,48 @@
+const filter = require('spam-filter')('naiveBayes');
+const textParser = require('./text-parser');
+// content refers to the contents of the blog post (article) sent by feedparser
+module.exports = async function (content) {
+  // Minimum words set to 20, can be changed
+  const MINWORDS = 5;
+
+  // Functions
+
+  /* function removeTags(str) { // Removes HTML tags from a string, leaving only plaintext
```

Comment:
Can we dump this commented out code?  You can add it back later in another PR

Reply:
I'll do it and rebase, so we can get this landed.

Reply:
Sorry, I just got home.
Thanks for removing it